nibabel/freesurfer/tests/test_io.py

```diff
@@ -92,18 +91,19 @@ def test_geometry():
         with clear_and_catch_warnings() as w:
             warnings.filterwarnings('always', category=DeprecationWarning)
             read_geometry(surf_path, read_metadata=True)
-        assert_true(any('volume information contained' in str(ww.message)
+        assert(any('volume information contained' in str(ww.message)
                         for ww in w))
-        assert_true(any('extension code' in str(ww.message) for ww in w))
+        assert(any('extension code' in str(ww.message) for ww in w))
         volume_info['head'] = [1, 2]
         with clear_and_catch_warnings() as w:
             write_geometry(surf_path, coords, faces, create_stamp, volume_info)
-        assert_true(any('Unknown extension' in str(ww.message) for ww in w))
+        assert(any('Unknown extension' in str(ww.message) for ww in w))
         volume_info['a'] = 0
-        assert_raises(ValueError, write_geometry, surf_path, coords,
-                      faces, create_stamp, volume_info)
+        with pytest.raises(ValueError):
+            write_geometry(surf_path, coords, faces, create_stamp, volume_info)
+        
 
-    assert_equal(create_stamp, read_create_stamp)
+    assert create_stamp == read_create_stamp
 
     np.testing.assert_array_equal(coords, coords2)
```

Comment:
I guess this is the convention you were following? Since `assert_array_equal` is imported, you can remove the `np.testing.` from all cases. If you want to have a consistent convention for the whole file, I would suggest settling on one of the following:

* `assert_array_equal(a, b)`
* `assert np.array_equal(a, b)`
* `assert np.all(a == b)`